pkg/controller/keycloak/keycloak_reconciler_test.go

```diff
@@ -0,0 +1,26 @@
+package keycloak
+
+import (
+	"github.com/keycloak/keycloak-operator/pkg/apis/keycloak/v1alpha1"
+	"github.com/keycloak/keycloak-operator/pkg/common"
+	"github.com/keycloak/keycloak-operator/pkg/model/keycloak"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestKeycloakReconciler_Test_Creating_Example_Service(t *testing.T) {
```

Comment:
Would be good to add an update test here as well.

Reply:
Done